store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -5710,6 +5710,15 @@ private void verifyPassword(Account acct, String password, AuthMechanism authMec
                         protocol = authCtxt.get(AuthContext.AC_PROTOCOL).toString();
                     }
                 }
+
+                boolean mCaptchaEnabled = acct.getBooleanAttr(Provisioning.A_zimbraCAPTCHAEnabled, false);
+                if (mCaptchaEnabled) {
+                    int loginFailCount = acct.getIntAttr(Provisioning.A_zimbraCAPTCHALoginFailedCount, 0);
+                    if (loginFailCount > 0 ) {
```

Comment:
again, the max failure count should be configurable.

Reply:
This functionality is going to be covered in new separate ticket